### ISSUE TITLE
Se requiere crear un código de conducta

### ISSUE THREAD

**Issue by laikmosh** [ProyectoPartidoMigala/organizacion-inicial#1]
Antes de iniciar ninguna discusión debemos definir un código de conducta, un código de conducta es un documento que establece las expectativas para el comportamiento de los participantes. 
**Construir y hacer cumplir un código de conducta puede ayudar a crear una atmósfera positiva y un espacio seguro de colaboración para esta comunidad.**
Establezcamos entre todos cuales son los comportamientos inaceptables y cómo debemos actuar ante un usuario que infringe las normas establecidas.

**Comment by crojasmrls** [comment]
Un buen punto de partida es el comunicado de sugerencias creada por la comisión de espacios seguros de la OPF.
![photo_2021-08-04_23-30-42](https://user-images.githubusercontent.com/37606633/128291283-1c82bc06-7968-4967-adba-a9f683078f9c.jpg)
![photo_2021-08-04_23-30-46](https://user-images.githubusercontent.com/37606633/128291288-71ec0687-9393-4777-9d37-788cf1a1f037.jpg)

**Comment by crojasmrls** [comment]
Otra lectura interesante es la definición de software libre y la filosofia que hay detras.
https://www.gnu.org/philosophy/free-sw.html
Creo que es bastante afin a los ideales del partido.

**Comment by laikmosh** [comment]
@crojasmrls Bueno si eso ya es oficial de la OPF creo que deberíamos incluirlo íntegramente, y sobre eso vamos agregando puntos ya más específicos al contexto de nuestra comisión, como lo que incluiste del software abierto

**Comment by crojasmrls** [comment]
Sí, de igual manera al no ser un documento final, podemos irlo actualizando conforme la comisión de espacios seguros lo vaya haciendo.

**Comment by laikmosh** [comment]
El punto 21 de ese código de conducta se refiere a "nosotras", no se si hay alguna razón por la que venga así y si es apropiado modificarlopor ejemplo a "nosotras y nosotros", o es correcto dejarlo como está
